examples/datasets/plot_limo_data.py

```diff
@@ -108,15 +106,11 @@
 # ----------------
 #
 # We can visualise the distribution of the face events contained in the
-# ``limo_epochs`` structure. Events should appear clearly grouped, as they
-# were ordered during the import process.
+# ``limo_epochs`` structure. Events should appear clearly grouped, as the
+# epochs are ordered by condition.
 
-fig, ax = plt.subplots(figsize=(7, 4))
-plot_events(limo_epochs.events, event_id=limo_epochs.event_id, axes=ax)
-ax.set(title="Distribution of events in LIMO epochs")
-plt.legend(loc='lower left', borderaxespad=1.)
-plt.tight_layout()
-plt.show()
```

Comment:
not necessary to call `plt.show()` in examples/tutorials. Also generally good to avoid `tight_layout`; it's already done internally (with some safety checks) for the plots where it makes sense.

Reply:
also, the legend gets added automatically if you let `plot_events` create the figure itself.